template/src/config/serverConfig.ts

```diff
@@ -0,0 +1,16 @@
+import { createLoader, values, safeValues } from 'configuru';
+
+console.log(`Loading the server configuration from ${process.env.CFG_JSON_PATH}`);
+
+const loader = createLoader({
+    defaultConfigPath: '.env.jsonc',
+});
+
+export const serverConfig = values({
+    // TODO: Add your server configuration here
+    myEnvVar: loader.string('MY_ENV_VAR'),
```

Comment:
do we need add all variables we add to the `.env.local.jsonc` also here? If sou it worths to mention it in the readme..

Reply:
I updated the docs 📝